Makefile

```diff
@@ -46,11 +46,12 @@ python-sqlalchemy-test: python-build
 ruby-test:
 	$(MAKE) -C languages/ruby test
 
+OSO_VERSION :=$(shell cat VERSION)
 java-test:
 	$(MAKE) -C languages/java package
 	cd test && \
-		javac -classpath "../languages/java/oso/target/*:." Test.java && \
-		java -classpath "../languages/java/oso/target/*:." -enableassertions Test
+		javac -classpath "../languages/java/oso/target/oso-$(OSO_VERSION).jar:." Test.java && \
```

Comment:
There are a lot of jars in target, shaded and unshaded versions of the library. This tests only against the one we release.